Cargo.toml

```diff
@@ -37,32 +27,8 @@ exclude = [
 ]
 
 [patch.crates-io]
-axum = { git = "https://github.com/grafbase/axum", rev = "c3146f9ca921907d9884fbf7549a1520e9e72eac" }                                 # axum-tungstenite-upgrade
 multipart-stream = { git = "https://github.com/grafbase/multipart-stream-rs-fork", rev = "06ff198e4041c8a8c1c93e580c260d597727c193" } # http-1.0-fix-multipart-mixed
```

Comment:
We have to upstream these changes. One is for HTTP 1.0 crate and another one a fix from graeme.